tests/common/test_logging.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) Microsoft Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from unittest import TestCase
+
+import pytest
+
+from playwright.async_api import Error, async_playwright
+from playwright.sync_api import sync_playwright
+
+
+def test_logging_sync(browser_name, launch_arguments):
+    class LoggingTestCase(TestCase):
+        def test_logging(self):
+            with self.assertLogs("playwright", level=logging.DEBUG) as logger:
+                with sync_playwright() as playwright:
+                    browser = getattr(playwright, browser_name).launch(
+                        **launch_arguments
+                    )
+                    page = browser.new_page()
+                    page.goto("https://example.com")
+                    with pytest.raises(Error):
+                        page.wait_for_selector("will never find", timeout=100)
+                    page.close()
+                    browser.close()
+            self.assertEqual(
+                logger.output,
+                [
+                    "DEBUG:playwright:=> browser_type.launch started",
```

Comment:
This is not how the pw:api looks typically, there are log entries inside it as well. I think Python users will be surprised not to see them:

<img width="739" alt="Screen Shot 2021-05-05 at 12 01 14 PM" src="https://user-images.githubusercontent.com/883973/117195050-aa147100-ad99-11eb-8008-174487cd68f6.png">

Reply:
This is how it is done in `playwright-java` too 
https://github.com/microsoft/playwright-java/blob/0fa416b45931a3db254665f84a5c8375edf67b58/playwright/src/main/java/com/microsoft/playwright/impl/LoggingSupport.java#L41 We can add more logging in another PR but let this one get merged first